configure

```diff
@@ -3730,6 +3730,7 @@ sr_filter_select="dnn"
 stereo3d_filter_deps="gpl"
 splitcc_filter_deps="avcodec"
 subscale_filter_deps="swscale avcodec"
+text2bitmap_filter_deps="libass"
```

Comment:
You are using libavformat and libavcodec, so you'll need to add them just like `subtitles_filter_deps`